### DISCUSSION TITLE
关于如何使用LobeChat对接非GPT类模型

### DISCUSSION THREAD

**Discussion by c4droid369** [lobehub/lobehub#2315] [Development Question] [answered]
想问一下怎么用LobeChat对接非GPT类的模型啊？我想用LobeChat作为demo前端来展示我训练的模型，关于对接模型的API有没有相关的文档或者示例什么的

**Comment by cy948** [comment, marked as answer]
libs/agent-runtime 下有各种provider，可以参考代码进行接入。

**Reply by c4droid369** [reply]
我刚刚看了libs/agent-runtime下面的源码，因为我拿到的api文档里面对于智能体和提示词模板都是通过两个api接口传过来的，套用的话需要做些什么操作

**Comment by arvinxx** [comment]
最简单的做法就是兼容 openai 的接口规范
